test/acceptance/features/bindAppToService.feature

```diff
@@ -305,3 +306,180 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-a-d-c" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-a-d-c" should be changed to "True"
         And Secret "binding-request-a-d-c" contains "SOME_KEY" key with value "SOME_VALUE:5432:db-demo-a-d-c"
+
+
+    Scenario: Creating binding secret from the definitions managed in OLM operator descriptors
+        Given Backend service "some-backend-service.v0.2.0" CSV is installed
```

Comment:
I would drop service name from the step - it can be obtained from the yaml content.

Reply:
done.